README.md

```diff
@@ -34,8 +38,8 @@ Use of header token for endpoints with security:
 |HTTP Verb|  Url Path                                 | Description                     | Comments                        |
 |:------- |:------------------------------------------|:------------------------------- |:--------------------------------|
 |  GET    | /nemory/api/v1.0/words                    |URL to read a collection of word | public access                   |
-|  GET    | /nemory/api/v1.0/words?word=term          |URL to read a single word filter by word|public access <br>query string: term <br> result ok: ![get words by term](./readme-images/response_ok_words_by_term.png?raw=true "Optional Title")  <br> not found the term: ![get words by term](./readme-images/response_not_found_term.png?raw=true "Optional Title") |
-|  GET    | /nemory/api/v1.0/words?category=verb      |URL to read a list of words filter by category |public access <br> query string: category  <br> result ok: ![get words by term](./readme-images/response_ok_by_category.png?raw=true "Optional Title") <br> not found the category: ![get words by term](./readme-images/response_not_found_term.png?raw=true "Optional Title") |
+|  GET    | /nemory/api/v1.0/word?word=term          |URL to read a single word filter by word|public access <br>query string: term <br> result ok: ![get words by term](./readme-images/response_ok_words_by_term.png?raw=true "Optional Title")  <br> not found the term: ![get words by term](./readme-images/response_not_found_term.png?raw=true "Optional Title") |
+|  GET    | /nemory/api/v1.0/word?category=verb      |URL to read a list of words filter by category |public access <br> query string: category  <br> result ok: ![get words by term](./readme-images/response_ok_by_category.png?raw=true "Optional Title") <br> not found the category: ![get words by term](./readme-images/response_not_found_term.png?raw=true "Optional Title") |
```

Comment:
I think this was a typo. Since word does not take category but word does.. Although fits more in word. Maybe would make more sense to add filtering for words than word actually